### PR TITLE
Update installation doc about Raspbian

### PR DESCRIPTION
**Pull request by daxian-dbw**
## PR Summary

Fix some typos in the doc and add Kali and Raspbian to the supported distros at the top.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [ ] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] User-facing [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed - Issue link:
- **Testing - New and feature**
    - [x] Not Applicable or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [ ] [Add `[feature]` if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

File: docs/installation/linux.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 Supports [Ubuntu 14.04][u14], [Ubuntu 16.04][u16], [Ubuntu 17.04][u17], [Debian 8][deb8], [Debian 9][deb9],
 [CentOS 7][cos], [Red Hat Enterprise Linux (RHEL) 7][rhel7], [OpenSUSE 42.2][opensuse], [Fedora 25][fed25],
-[Fedora 26][fed26], and [Arch Linux][arch].
+[Fedora 26][fed26], [Arch Linux][arch], [Kali Linux][kali] and [Raspbian][rasp].
```
Comment (iSazonov) [inline docs/installation/linux.md]: Should we remove unsupported (EOL) distributives (Ubuntu 17.04, OpenSUSE 42.2, Fedora 25)?

Comment (daxian-dbw) [reply docs/installation/linux.md]: I think we should update the content about `Ubuntu 17.04, OpenSUSE 42.2, Fedora 25` during the preview.3 release. Now we still have old packages (17.04 and etc.) in the existing release pages, but in preivew.3, we will release `17.10` package.

@rjmholt Did you make any packigng changes for the RPM package? (anything we nee to do to support Fedora 27?)

Comment (rjmholt) [reply docs/installation/linux.md]: No updates to the RPM -- it all worked fine as is

Comment (daxian-dbw) [reply docs/installation/linux.md]: Open #6867 to track this work during the next release.

**Review by anmenaga — APPROVED** [review]

File: docs/installation/linux.md (hunk 1 above)
Comment (rjmholt) [reply docs/installation/linux.md]: The other thought I had when looking at new release packaging was the the URLs we use for Ubuntu packages should be updated.

Comment (daxian-dbw) [reply docs/installation/linux.md]: @rjmholt what URLs? Can you please update #6867 with that information?

**Pull request merged**